great_tables/_render_checks.py

```diff
@@ -0,0 +1,38 @@
+import warnings
+
+from ._gt_data import GTData
+from ._render import infer_render_env
+
+
+class RenderWarning(Warning):
+    """Base warning for render checks."""
+
+
+def _render_check(data: GTData):
+    if infer_render_env() == "quarto":
+        _render_check_quarto(data)
+
+
+def _render_check_quarto(data: GTData):
+    """Check for rendering issues in Quarto.
+
+    * Quarto uses Pandoc internally to handle tables, and Pandoc tables do not support pixel widths.
+      As a result, when cols_width is used in a Quarto environment, widths need to be converted to
+      percentages.
+    * Alternatively, users may set the option quarto_disable_processing to True.
+    * Disabling table processing also helps with pieces like table striping, but means Quarto will
+      not process cross-references, etc..
+    """
+
+    # quarto_disable_processing is set, no need to warn ----
+    if data._options["quarto_disable_processing"]:
+        return
+
+    # cols_widths set ----
+    if any([col.column_width is not None for col in data._boxhead]):
```

Comment:
The width configuration still works fine in Pandoc if it's given in percentages (which is why I wanted to do the fixup in the previous PR as I did). It's only when they're given in pixels that Pandoc doesn't like to parse them.

If I understand the code correctly here, the warning you're giving is maybe too generic, since it will warn in cases where things would still work fine.

Reply:
Ah, thanks! I'll add a piece that skips the warning if the user puts only percentages, and clarify when I warn that it's because non-percentages were used...